kaldb/src/main/java/com/slack/kaldb/metadata/core/UpdatableCacheableMetadataStore.java

```diff
@@ -40,7 +40,8 @@ public void updateSync(T metadataNode) {
     try {
       update(metadataNode).get(KaldbConfig.DEFAULT_ZK_TIMEOUT_SECS, TimeUnit.SECONDS);
     } catch (InterruptedException | ExecutionException | TimeoutException e) {
-      throw new InternalMetadataStoreException("Error updating node: " + metadataNode, e);
+      logger.error("Error updating znode: " + metadataNode, e);
```

Comment:
Can we do this without the additional logs before throwing the exceptions? When every place does this up the stack trace it makes it difficult to understand, as there's a lot of duplicate error logs. It also skews the metrics as we count these multiple times.

Reply:
I'll just revert the last commit - I guess I was okay with duplicate logs because not all callers can be audited to make sure the exception is being propagated but seeing multiple errors is inconvenient